src/handlers/misc/Roles.ts

```diff
@@ -0,0 +1,26 @@
+import { Message, GuildMemberRoleManager } from 'discord.js'
+import { User } from '../../entity/User'
+import AbstractHandler from '../abstract/AbstractHandler'
+import { Command } from '../../entity/Command'
+
+export default class RoleHandler extends AbstractHandler {
+    public constructor() {
+        super(false, false, false, false)
+    }
+
+    protected async handler(user: User, cmd: Command, msg: Message): Promise<any> {
+        const userRoles: GuildMemberRoleManager = await msg.member.roles
+        const raconteur = '799498542641053726' // this should refer to the role in config.ts 0_0
```

Comment:
DM me on Discord, and I'll send you the `config.ts` file so that you're able to change config and refer to it in the code.

Reply:
Okay so I got your reply just now. I've updated config.